app/controllers/legacy/users_controller.rb

```diff
@@ -31,12 +31,6 @@ def update
 
     private
 
-    def prevent_caching
```

Comment:
why was only this removed from the legacy user controller? (I realize none of it should be getting used anymore)

Reply:
👍 . Addressed here: https://github.com/openstax/accounts/pull/859#discussion_r450941766